src/domain/token/Token.ts

```diff
@@ -104,42 +105,63 @@ export class Token {
 
     if (options.tradeDirection == 'BUY') {
       if (options.fixedSide == FixedSide.IN) {
+        const tokenAmountWithSlippage =
```

Comment:
don't you think it's cleaner to have bps converter instead of doing the division everywhere? a small util function could do

Reply:
I've created util functions for applying slippage. Creating bps converter did not really work that well because the amount (token/collateral) is a BigInt and if I just did the division I would not be able to cast decimals into a BigInt like so `tokenAmount / BigInt(convertSlippageBps(slippageBps))` as this would result in `tokenAmount / BigInt(0.05`